tabular/src/autogluon/tabular/configs/config_helper.py

```diff
@@ -0,0 +1,368 @@
+        self.config['hyperparameter_tune_kwargs'] = hyperparameter_tune_kwargs
+
+        return self
+
+    def ag_args(self, ag_args: dict) -> ConfigBuilder:
+        """
+        Keyword arguments to pass to all models (i.e. common hyperparameters shared by all AutoGluon models).
+        See the `ag_args` argument from "Advanced functionality: Custom AutoGluon model arguments" in the `hyperparameters` argument documentation for valid values.
+        Identical to specifying `ag_args` parameter for all models in `hyperparameters`.
+        If a key in `ag_args` is already specified for a model in `hyperparameters`, it will not be altered through this argument.
+        """
+        self.config['ag_args'] = ag_args
+        return self
+
+    def ag_args_fit(self, ag_args_fit: dict) -> ConfigBuilder:
+        """
+        Keyword arguments to pass to all models.
+        See the `ag_args_fit` argument from "Advanced functionality: Custom AutoGluon model arguments" in the `hyperparameters` argument documentation for valid values.
+        Identical to specifying `ag_args_fit` parameter for all models in `hyperparameters`.
+        If a key in `ag_args_fit` is already specified for a model in `hyperparameters`, it will not be altered through this argument.
+        """
+        self.config['ag_args_fit'] = ag_args_fit
+        return self
+
+    def ag_args_ensemble(self, ag_args_ensemble: dict) -> ConfigBuilder:
+        """
+        Keyword arguments to pass to all models.
+        See the `ag_args_ensemble` argument from "Advanced functionality: Custom AutoGluon model arguments" in the `hyperparameters` argument documentation for valid values.
+        Identical to specifying `ag_args_ensemble` parameter for all models in `hyperparameters`.
+        If a key in `ag_args_ensemble` is already specified for a model in `hyperparameters`, it will not be altered through this argument.
+        """
+        self.config['ag_args_ensemble'] = ag_args_ensemble
+        return self
+
+    def excluded_model_types(self, models: Union[str, list]) -> ConfigBuilder:
+        """
+        Banned subset of model types to avoid training during `fit()`, even if present in `hyperparameters`.
+        Reference `hyperparameters` documentation for what models correspond to each value.
+        Useful when a particular model type such as 'KNN' or 'custom' is not desired but altering the `hyperparameters` dictionary is difficult or time-consuming.
+            Example: To exclude both 'KNN' and 'custom' models, specify `excluded_model_types=['KNN', 'custom']`.
+        """
+        valid_keys = [m for m in MODEL_TYPES.keys() if m not in ['ENS_WEIGHTED', 'SIMPLE_ENS_WEIGHTED']]
+        if not isinstance(models, list):
+            models = [models]
+        for model in models:
+            assert model in valid_keys, f'{model} is not one of the valid models {valid_keys}'
+        self.config['excluded_model_types'] = sorted(list(set(models)))
```

Comment:
why sort it?

Reply:
This is for the test stability. `set` order is non-deterministic and `['LR', 'NN']` should not be treated differently from `['NN', 'LR']` in this case.